cumulusci/tasks/salesforce/users/permsets.py

```diff
@@ -0,0 +1,53 @@
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+from cumulusci.core.exceptions import CumulusCIException
+from cumulusci.core.utils import process_list_arg
+
+
+class AssignPermissionSets(BaseSalesforceApiTask):
+    task_options = {
+        "api_names": {
+            "description": "API names of desired Permission Sets, separated by commas.",
+            "required": True,
+        },
+    }
+
+    def _init_options(self, kwargs):
+        super()._init_options(kwargs)
+
+        self.options["api_names"] = process_list_arg(self.options["api_names"])
+
+    def _run_task(self):
+        # Determine existing assignments
+        query = f"""SELECT Id,
+                           (SELECT PermissionSetId
+                            FROM PermissionSetAssignments)
+                    FROM User
+                    WHERE Id = '{self.org_config.user_id}'"""
```

Comment:
For scratch orgs, `user_id` is a property which does its own query of User filtering by username. Would it make more sense to just filter by username in this query?